### PR TITLE
fix script for models repo stable

### LINKED ISSUE #28: 自动稳定性验证
监测model repo的改动， 自动触发变更的model的稳定运行， 通过多次运行结果判断model的变更是否符合预期。 
teamcity 地址：
http://18.222.34.7:8080/viewType.html?buildTypeId=Paddle_CeTaskEvaluation

### PR DESCRIPTION
**Pull request by guochaorong**
resolve #28

### CODE REVIEW

File: eva.xsh [hunk 1]
```diff
@@ -40,13 +31,14 @@ def get_changed_tasks(args):
     if args.task_dir:
         tasks = args.task_dir.split()
         return tasks
-    ret, out, err = RunCmd('''cd tasks; git diff HEAD^ HEAD | grep "diff --git" | awk -F' ' {'print $4'}''')
-    print (ret,out,err)
-    out = out.strip().decode('utf-8')
+    cd @(config.baseline_path)
+    out = $(git diff HEAD^ HEAD | grep "diff --git")
```
Comment (Superjomn) [inline eva.xsh]: git diff develop
跟基线比

File: eva.xsh [hunk 2]
```diff
@@ -40,13 +31,14 @@ def get_changed_tasks(args):
...
+    out = out.strip()
     for item in out.split('\n'):
-        task = item.split('/')[1]
+        task = item.split()[3].split('/')[1]
         if task not in tasks:
             tasks.append(task)
+    print (tasks)
```
Comment (Superjomn) [inline eva.xsh]: log.warn

File: analysis_kpis.py [hunk 3]
```diff
@@ -51,7 +51,7 @@ def print_result(self):
                 suc = False
                 print("kpi: %s change_tate too bigger !!!!!!!!!!" % kpi_name)
             else:
-                print('kpi:%s' % kpi_name)
+                print('kpi: %s' % kpi_name)
```
Comment (Superjomn) [inline analysis_kpis.py]: log.warn

File: eva.xsh (hunk 2 above)
Comment (guochaorong) [reply eva.xsh]: log.info is better?

File: eva.xsh (hunk 1 above)
Comment (guochaorong) [reply eva.xsh]: done

**Pull request closed without merging**